spec/features/team/show_team_spec.rb

```diff
@@ -61,4 +61,19 @@
       expect(page).to have_link(I18n.t("helpers.links.email"), href: mailto_link_for_user)
     end
   end
+
+  describe 'display team members only for logged in users' do
+
+    it 'should not display team member table for users who are not logged in ' do
+      sign_out @user
+      visit team_path @team
+      expect(page).to_not have_css("table.table-striped")
```

Comment:
@tomkellog May not be the best approach to check on a table with no specific id. Imagine there are going to be multiple tables with the css-class `.table-striped` on this page... What about giving the table a meaningful id?